scripts/windows/setup/Install-SecurityDaemon.ps1

```diff
@@ -59,9 +68,23 @@ function Install-SecurityDaemon {
 function Test-IsDockerRunning {
     if ((Get-Service "Docker").Status -eq "Running") {
         Write-Host "Docker is running." -ForegroundColor "Green"
-        return $true
-    }
-    else {
+        $os = Invoke-Native "docker version --format {{.Server.Os}}" -Passthru
+        if (($UseWindowsContainers) -and -not ($os -match "\s*windows\s*$")) {
```

Comment:
On Windows Server Core 1803 this won't work because there's no `dockercli.exe`; only Windows containers are supported. So we need to be able to detect that and display the right error message.

Reply:
Added logic to handle this.